### PR TITLE
fix rosie REST API docs

### PR DESCRIPTION
**Pull request by oliver-sanders**
Fix a couple of broken examples in the docs.

### CODE REVIEW

File: sphinx/api/rosie-web.rst [hunk 1]
```diff
@@ -152,7 +152,7 @@ REST API
 
       .. code-block:: http
 
-         GET http://host/my_prefix/search?var+bob+nowcast&format=json HTTP/1.1
+         GET http://host/my_prefix/search/var+bob+nowcast&format=json HTTP/1.1
```
Comment (matthewrmshin) [inline sphinx/api/rosie-web.rst]: Why doesn't the old syntax work any more?

Comment (oliver-sanders) [reply sphinx/api/rosie-web.rst]: I don't think this syntax was ever correct, was it?

Comment (matthewrmshin) [reply sphinx/api/rosie-web.rst]: That's true.

Comment (oliver-sanders) [reply sphinx/api/rosie-web.rst]: It's probably either an error copied forward from the old docs or a translation error by yours truely :frowning:

Comment (sadielbartholomew) [reply sphinx/api/rosie-web.rst]: Isn't this meant to instead be:
```suggestion
         GET http://host/my_prefix/search?s=var+bob+nowcast&format=json HTTP/1.1
```

The original, as with your change suggestion, both don't make sense as URL queries, but this does? To check I am not completely mistaken, I tried these URLs out on both the current ("old") & my new (PR #2288) versions of the rosie discovery utility, & the original & change here both give a 404 error.

**Comment by oliver-sanders** [comment]
> The original & change here both give a 404 error

The change doesn't give a 404, for example:

http://fcm1/rosie/mot/search/frsn

However this isn't the user facing page not the REST API! Whoops! `s` is what we were supposed to document here:

http://fcm1/rosie/mot/search?s=frsn

**Comment by oliver-sanders** [comment]
Yep, that should be the last of them.

**Pull request merged**